packages/testing/src/testProxy.options.ts

```diff
@@ -0,0 +1,47 @@
+import { binaryMimeTypes, getHeaderByKey, ServerProxyResponse } from '@marblejs/proxy';
+import { HttpHeaders, HttpMethod } from '@marblejs/core';
+
+export interface TestRequest {
+  method: HttpMethod;
+  path: string;
+  host?: string;
+  protocol?: string;
+  headers: HttpHeaders;
+  body?: any;
+}
+
+export interface TestResponse {
+  statusCode: number;
+  statusMessage?: string;
+  body: any;
+  headers: Record<string, string[]>;
+}
+
+export type BodyParser = (response: ServerProxyResponse) => any;
+
+export interface TestProxyOptions {
+  bodyParser: BodyParser;
+}
+
+export const defaultBodyParser: BodyParser = response => {
```

Comment:
Can we reuse `defaultParser` from `@marblejs/middleware-body`?

Reply:
I'm afraid that the implementations will not match at some point...

Reply:
Yup, it seems to be the best solution.